common.js

```diff
@@ -2716,6 +2716,44 @@ const allSoloCars = {
         '2004': ['ep'],
     },
   },
+  'Scion': {
+    'FR-S (incl. Release Series 1.0 and 2.0)': {
+        'all': ['ssc','stx','csp'],
+    },
+    'FR-S *Limited Prep*': {
+      'all': ['dsp'],
+    },
+    'FR-S (with TRD suspension; incl. Release Series 1.0 and 2.0)': {
+      'all': ['cs'],
```

Comment:
Please add an entry for each year (2013-2016) and also be sure to add `ssc` as a valid class.